deploy/ansible/roles/local/ansible-users/meta/main.yml

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
this file looks like redundant, should it be gitignored?

Reply:
this file is a part of an external module. Some functionality could be added in future